hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/server/JournaledEditsCache.java

```diff
@@ -277,11 +279,10 @@ void storeEdits(byte[] inputData, long newStartTxn, long newEndTxn,
         initialize(INVALID_TXN_ID);
         Journal.LOG.warn(String.format("A single batch of edits was too " +
                 "large to fit into the cache: startTxn = %d, endTxn = %d, " +
-                "input length = %d. The capacity of the cache (%s) must be " +
+                "input length = %d. The capacity of the cache must be " +
```

Comment:
Can we keep the key here in the error message, but print both? Like `(%s or %s)` where one is `DFS_JOURNALNODE_EDIT_CACHE_SIZE_KEY` and one is `DFS_JOURNALNODE_EDIT_CACHE_SIZE_FRACTION_KEY`